react/features/shared-video/functions.ts

```diff
@@ -146,3 +148,31 @@ export function isURLAllowedForSharedVideo(url: string,
 
     return false;
 }
+
+/* eslint-disable max-params */
```

Comment:
Do we need this?

The parameter is 1. Isn't it?

Reply:
I moved it with the command, seems someone forgot to remove it at some point.